src/app/endpoints/page.jsx

```diff
@@ -0,0 +1,143 @@
+"use client";
+import Footer from "../components/Footer";
+import Nav from "../components/nav";
+import React, { useState } from "react";
+import Link from "next/link";
+
+export default function Endpoints() {
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  const toggleTheme = () => {
+    setIsDarkMode(!isDarkMode);
+  };
+
+  const sectionStyle = "p-4 text-left w-full max-w-[26rem] md:max-w-[40rem] bg-white border border-gray-200 rounded-lg shadow sm:p-8 dark:bg-gray-800 dark:border-gray-700";
+
+  return (
+    <main
+      className={`${isDarkMode ? "bg-gray-900 text-gray-400" : "bg-gray-100 text-gray-800"} min-h-screen pb-10`}
+    >
+      <Nav toggleTheme={toggleTheme} isDarkMode={isDarkMode} />
```

Comment:
Use `NavBar` instead

Reply:
just saw this.  I'll change that as well